Server/ReasnAPI/ReasnAPI/Controllers/TestController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ReasnAPI.Models.DTOs;
+using ReasnAPI.Services;
+
+namespace ReasnAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TestController : Controller
+    {
+        
+        private readonly TagService _tagService;
+
+        public TestController(TagService tagService)
+        {
+            _tagService = tagService;
+        }
+
+        [HttpGet]
+        public IActionResult Add()
+        {
+
+            var tagDto = new TagDto
+            {
+                Name = "TestTagasd"
+            };
+
+            var result = _tagService.CreateTag(tagDto);
+
+            return Ok(result);
+        }
+    }
+}

```

Comment:
Those 'test' ones shouldnt really be commited, you can leave it locally, but I assume you forgote to exclude it

Reply:
Right